backend/main.py

```
@@ -908,32 +929,91 @@ async def get_current_user_profile(current_user: User = Depends(get_current_user
+        "cookie_settings": {
+            "secure": settings.environment == "production",
+            "samesite": "none" if settings.environment == "production" else "lax",
+            "httponly": True
+        },
+        "cors_check": "See response headers for Access-Control-Allow-Credentials"
+    }
+
 @app.post("/test-login")
 async def test_login(
     user: UserLogin, 
     request: Request,
```

Ruff (0.14.3)

Unused function argument: `request`

(ARG001)